### PR TITLE
Prod 16 modal

### PR DESCRIPTION
**Pull request by Valdas-Stonkus**
Created two components, Modal and Button.

### CODE REVIEW

**Review by MarcinkEve — APPROVED** [review]

File: src/Components/Modal/Modal.scss [hunk 1]
```diff
@@ -0,0 +1,139 @@
+$whiteColor: #fff;
```
Comment (SwallowEve) [inline src/Components/Modal/Modal.scss:1]: Valdas, when my branch will be merged, you'll find in main branch assets folder with _variables.scss file where all variables are stored globally. If I'm not mistaken these variables should be moved there then too.

File: src/Components/Modal/Modal.jsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        document.querySelector('.modal-dialog').classList.add('modal-dialog--close')
+        document.querySelector('.modal').classList.add('modal--close')
+        setTimeout(() => {
+            action()
+        }, 300)
+    }
+
+    return (
+        <div className="modal fade" tabIndex="-1" role="dialog" aria-labelledby="ModalScrollableTitle" aria-hidden="true" onClick={() => handleClose(onClose)}>
+            <div className="modal-dialog modal-dialog-scrollable" role="document">
```
Comment (SwallowEve) [inline src/Components/Modal/Modal.jsx:35]: Shouldn't it be here modal__dialog, modal__header and etc. according to Bem?

**Review by SwallowEve — APPROVED** [review]
I added two comments in modal.sccs. Nice done!

File: src/Components/Modal/Modal.jsx (hunk 2 above)
Comment (Valdas-Stonkus) [reply src/Components/Modal/Modal.jsx:35]: By BEM yes, but here I use clases of Bootstrap with my unique prefix cust-. So any who will use component can take class name from Bootstrap website add suffix and use it in his code

File: src/Components/Modal/Modal.scss (hunk 1 above)
Comment (Valdas-Stonkus) [reply src/Components/Modal/Modal.scss:1]: Yes, we could do it if we will think that it need to be do.

File: src/Components/Modal/Modal.jsx (hunk 2 above)
Comment (Valdas-Stonkus) [reply src/Components/Modal/Modal.jsx:35]: There is no effect for BEM stylings.

**Pull request merged**